spinnaker-monitoring-daemon/spinnaker-monitoring/http_server.py

```diff
@@ -113,8 +113,8 @@ def get_option(name):
       """Return command-line option override or configuration value"""
       value = options.get(name)
       return value if value is not None else server_options[name]
-    port = get_option('port')
-    host = get_option('host')
+    port = get_option('port') or 8008
+    host = get_option('host') or ''
```

Comment:
For clarity, I would default to `0.0.0.0`. Given that in Python I think empty string is the same thing as 0.0.0.0, it would at least make the log statement on 119 make more sense.